### ISSUE TITLE
The official website is down, and an error is reported

### ISSUE THREAD

**Issue by hexWars** [umami-software/umami#1898]
```
yarn run v1.22.19
$ prisma migrate deploy
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": MySQL database "umami" at "rm-bp12k5m7wx3jrs06oeo.mysql.rds.aliyuncs.com:3306"

1 migration found in prisma/migrations

Applying migration `01_init`
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 01_init

Database error code: 1071

Database error:
Specified key was too long; max key length is 767 bytes

Please check the query number 1 from the migration file.


error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

**Comment by bboysoulcn** [comment]
> ```
> yarn run v1.22.19
> $ prisma migrate deploy
> Environment variables loaded from .env
> Prisma schema loaded from prisma/schema.prisma
> Datasource "db": MySQL database "umami" at "rm-bp12k5m7wx3jrs06oeo.mysql.rds.aliyuncs.com:3306"
> 
> 1 migration found in prisma/migrations
> 
> Applying migration `01_init`
> Error: P3018
> 
> A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
> 
> Migration name: 01_init
> 
> Database error code: 1071
> 
> Database error:
> Specified key was too long; max key length is 767 bytes
> 
> Please check the query number 1 from the migration file.
> 
> 
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ```

可能是索引的问题,rds地址露出来了

**Comment by hexWars** [comment]
> > ```
> > yarn run v1.22.19
> > $ prisma migrate deploy
> > Environment variables loaded from .env
> > Prisma schema loaded from prisma/schema.prisma
> > Datasource "db": MySQL database "umami" at "rm-bp12k5m7wx3jrs06oeo.mysql.rds.aliyuncs.com:3306"
> > 
> > 1 migration found in prisma/migrations
> > 
> > Applying migration `01_init`
> > Error: P3018
> > 
> > A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
> > 
> > Migration name: 01_init
> > 
> > Database error code: 1071
> > 
> > Database error:
> > Specified key was too long; max key length is 767 bytes
> > 
> > Please check the query number 1 from the migration file.
> > 
> > 
> > error Command failed with exit code 1.
> > info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> > ```
> 
> 可能是索引的问题,rds地址露出来了

ok，谢谢，那个也快过期了没事

**Comment by franciscao633** [comment]
I think this has been fixed in MySQL versions 5.7 and up. I'm going to bump up the version check in check-db.js to 5.7. Upgrade your version if possible
